published/external/afxdp.h

```diff
@@ -3,15 +3,6 @@
 // Licensed under the MIT License.
 //
 
-//
```

Comment:
I have mixed feelings about removing all the comments from the C files. On one hand I definitely don't want duplication, and I want the real docs to be the source of truth. On the other hand, I do feel it's good to have some information in the headers.

Reply:
I've left a few critical comments in headers (e.g. "Reserved") but in general I believe duplicating descriptive comments is more harmful than helpful, as it doubles the maintenance burden and also the developer burden if either the header or markdown documentation is considered "incomplete".